src/Setup/Condition/PHPExtensionLoadedCondition.php

```diff
@@ -10,12 +10,14 @@ class PHPExtensionLoadedCondition extends ExternalConditionObjective
 {
     public function __construct($which)
     {
+        $ilias_version = ILIAS_VERSION_NUMERIC;
+
         return parent::__construct(
             "PHP extension \"$which\" loaded",
             function (Setup\Environment $env) use ($which) : bool {
                 return in_array($which, get_loaded_extensions());
             },
-            "ILIAS 7 requires the PHP extension $which."
+            "ILIAS $ilias_version ILIAS_VERSION_NUMERIC . " . " requires the PHP extension $which."
```

Comment:
Could it be that `ILIAS_VERSION_NUMERIC` was slipped in the string here?

Reply:
@PurHur What do you mean :)?

Reply:
Sorry i forgot to click submit.